packages/app/src/app/login.tsx

```
@@ -56,6 +61,10 @@ const TitleContainer = styled.View`
   width: 100%;
 `;
 
+const ContentContainer = styled.View`
```

ESLint

[error] 'ContentContainer' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)